Source/WebCore/platform/audio/AudioArray.h

```diff
@@ -102,7 +102,7 @@ class AudioArray {
     T& operator[](size_t i) { return at(i); }
     const T& operator[](size_t i) const { return at(i); }
 
-    void zero() { memsetSpan(this->span(), 0); }
+    void zero() { zeroSpan(this->span()); }
```

Comment:
Why is `this->` needed?

Reply:
No, I've dropped a few `this->` from this header.